docs/ReferenceManyToManyField.md

```diff
@@ -20,18 +22,17 @@ import {
 } from 'react-admin';
 import { ReferenceManyToManyField } from '@react-admin/ra-relationships';
 
-export const BookShow = () => (
+export const BandShow = () => (
     <Show>
         <SimpleShowLayout>
-            <TextField source="title" />
-            <DateField source="publication_date" />
+            <TextField source="name" />
             <ReferenceManyToManyField
-                reference="authors"
-                through="book_authors"
-                using="book_id,author_id"
+                reference="venues"
+                through="performances"
+                using="band_id,venue_id"
```

Comment:
you should add a  `label="Performances"` as in the Input version... and redo the screenshot.